cis/Issues/digitalInsurancePassport/description.xml

```diff
@@ -0,0 +1,130 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Digital Insurance Passport</en>
+		<nl>Demo Digitaal Paspoort Verzekeren</nl>
+	</Name>
+	<ShortName>
+		<en>Demo Digital Insurance Passport</en>
+		<nl>Demo Digitaal Paspoort Verzekeren</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>cis</IssuerID>
+	<CredentialID>digitalInsurancePassport</CredentialID>
+	<Description>
+		<en>With your Digital Insurance Passport, insurers can see who you are and determine you do not appear in national and international sanctions, investigation and incident registers. This check is mandatory by law and insurers do these together to prevent costs and extra work for customers and insurers.</en>
+		<nl>Met jouw Digitaal Paspoort Verzekeren kunnen verzekeraars zien wie je bent en dat je niet voorkomt in nationale en internationale sanctie-, opsporings- en incidentregisters. Deze controle is wettelijk verplicht en doen verzekeraars samen om kosten en extra werk te voorkomen voor klanten en verzekeraars.</nl>
+	</Description>
+	<ShouldBeSingleton>true</ShouldBeSingleton>
+	<IssueURL>
+		<en>https://proddfppfademo.azurewebsites.net/</en>
+		<nl>https://proddfppfademo.azurewebsites.net/</nl>
+	</IssueURL>
+
+	<!-- <ForegroundColor>default</ForegroundColor>
+	<BackgroundGradientStart>default</BackgroundGradientStart>
+	<BackgroundGradientEnd>default</BackgroundGradientEnd> -->
+	<IsInCredentialStore>true</IsInCredentialStore>
```

Comment:
We don't allow demo credentials to be part of the credential overview in the IRMA-app.

Reply:
I disabled the feature by setting the setting to false.